corehq/toggles/__init__.py

```diff
@@ -2378,3 +2378,14 @@ def _commtrackify(domain_name, toggle_is_enabled):
     namespaces=[NAMESPACE_DOMAIN],
     description='This feature provides the release logs for application.'
 )
+
+TABLEAU_USER_SYNCING = StaticToggle(
```

Comment:
Should this depend on the `embedded_tableau` flag, or is it separate enough that it could be enabled on its own?

Reply:
I think @nospame means the reverse, would anyone ever use the user syncing without the embedded Tableau flag? If not you could add the embedded Tableau flag as a 'parent toggle'.